### ISSUE TITLE
Widget causing Tortoisegit to choke on commit owing to /path / pathfile being mutually exclusive

### ISSUE THREAD

**Issue by Venomed** [paulcbetts/SaveAllTheTime#46]
I'm using TortoiseGit 1.8.16.0 with git version 2.6.4.windows.1

Everytime I click the widget to commit I'm presented with the error:

![image](https://cloud.githubusercontent.com/assets/808099/12119839/dd99bc98-b3c5-11e5-9ba3-5b80b5e532d3.png)

Any ideas on how I can work around this or if it's a quick fix, I'll just wait for that :)

**Comment by paulcbetts** [comment]
Clicking the widget only works with GitHub Desktop, you should use that. If TortoiseGit is opening, they've co-opted it and are being Bad(tm) /cc @shiftkey

**Comment by Venomed** [comment]
Oh, didn't realise it was Github Desktop only. I used this extension approximately 2 years ago and it used to work perfectly (back when it was causing EF4 to generate designer files over and over) so I thought may be something had broken on this end. TortoiseGit did offer me github:// support during installation and I opted into that. I guess I should open an issue there then? (assuming github:// protocol dictates /path /pathfile combo to be valid?)
